family_context/urls.py

```diff
@@ -7,10 +7,14 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("core.urls")),
-    path("accounts/", include("allauth.urls")),
     path("home/", Home.as_view(), name="home"),
 ]
 
+if settings.SSO_USED:
+    urlpatterns.append(path("accounts/", include("allauth.urls")))
+else:
+    urlpatterns.append(path("accounts/", include("django.contrib.auth.urls")))
+
```

Comment:
manage.py runserver now returns the error d`jango.urls.exceptions.NoReverseMatch: Reverse for 'account_logout' not found. 'account_logout' is not a valid view function or pattern name.` when I attempt to sign in.

I've confirmed that the else block here is run so the auth.urls should be recognised. However, they seem not to be. 
Does it run fully on your end?

Reply:
I have fixed it so that it handles logout redirecting. Could you have a try for me?